models/artwork.coffee

```diff
@@ -103,7 +103,7 @@ module.exports = class Artwork extends Backbone.Model
   #
   isContactable: ->
     return false if @isAtLimitedFairPartner()
-    return true if @get('is_inquireable')
+    return true if @get('inquireable') or @get('is_inquireable')
```

Comment:
This is still used when casting metaphysics queries to Backbone models, but it's also used in the deprecated columsn/filter views   😐 

This makes the 'Contact Gallery' link correctly show up.